x-kiosk-imx8/Dockerfile

```diff
@@ -0,0 +1,225 @@
+
+COPY 6e12c13c0165c94615928ca3d9692e91cf0619c4.patch .
+RUN patch -p1 < 6e12c13c0165c94615928ca3d9692e91cf0619c4.patch 
+
+# chromium-wayland: https://github.com/OSSystems/meta-browser/tree/master/meta-chromium/recipes-browser/chromium/files/chromium-wayland
+COPY 0001-ozone-add-va-api-support-to-wayland.patch .
+COPY 0001-ozone-wayland-fix-re-initialization-of-WBMG.patch .
+COPY 0001-ozone-wayland-fixed-terminate-caused-by-binding-to-wrong-version.patch .
+
+RUN patch -p1 < 0001-ozone-add-va-api-support-to-wayland.patch
+RUN patch -p1 < 0001-ozone-wayland-fix-re-initialization-of-WBMG.patch
+RUN patch -p1 < 0001-ozone-wayland-fixed-terminate-caused-by-binding-to-wrong-version.patch
+
+#https://github.com/OSSystems/meta-browser/tree/master/meta-chromium/recipes-browser/chromium/files
+COPY 0007-Delete-compiler-options-not-available-in-release-ver.patch .
+COPY 0014-ozone-wayland-don-t-build-xcb-for-pure-wayland-build.patch .
+COPY 0015-IWYU-add-memory-for-std-unique_ptr-in-base-CommandLi.patch .
+
+RUN patch -p1 < 0007-Delete-compiler-options-not-available-in-release-ver.patch
+RUN patch -p1 < 0014-ozone-wayland-don-t-build-xcb-for-pure-wayland-build.patch
+RUN patch -p1 < 0015-IWYU-add-memory-for-std-unique_ptr-in-base-CommandLi.patch
+
+RUN autoninja -C out/Default/ chrome
+
+FROM ubuntu:20.04 
+
+LABEL maintainer="Raul Munoz <raul@foundries.io>"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV WAYLAND_USER="weston"
+ENV XDG_RUNTIME_DIR="/run/user/63"
+ENV WAYLAND_DISPLAY="wayland-0"
+
+ENV FIO_HOME /home/weston
+ENV FIO_USERID 63
+ENV FIO_USERNAME weston
+
+RUN groupadd -g 61 render
+
+RUN if [ ! "${FIO_USERID}" = "0" ]; then \
+        groupadd -g ${FIO_USERID} ${FIO_USERNAME}; \
+        useradd -d ${FIO_HOME} -s /bin/bash -m ${FIO_USERNAME} -u ${FIO_USERID} -g ${FIO_USERID} -G video,render; \
+    fi
+
+RUN apt-get update && \
+    apt-get install -y vim curl wget libglib2.0-dev libnss3 libatk1.0-dev \
+                       libatk-bridge2.0-0 libjpeg-turbo8 libdrm-dev \
+                       libxkbcommon0 libasound2 libflac8 libxslt1.1 \
+                       fontconfig-config libwayland-cursor0 libwayland-client0 \
+                       libwayland-egl1 libxdamage1 libxfixes3 strace && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -y update && apt-get install -y --no-install-recommends \
+    libwayland-client0 \
+    libwayland-server0 \
+    && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
+
+COPY libgal-imx_6.4.3.p1.4-aarch64-r0_arm64.deb /
+COPY libg2d1_6.4.3.p1.4-r0_arm64.deb /
+COPY libdrm2_2.4.102.imx-r0_arm64.deb /
+COPY imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb /
+
+RUN \
+    dpkg -i libgal-imx_6.4.3.p1.4-aarch64-r0_arm64.deb && \
+    dpkg -i libg2d1_6.4.3.p1.4-r0_arm64.deb && \
+    dpkg -i libdrm2_2.4.102.imx-r0_arm64.deb  && \
+    dpkg -i imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb
+
+RUN \
+    mkdir -p /usr/lib/chromium/ && \
```

Comment:
this command doesn't need because of following one on line 199 `mkdir -p /usr/lib/chromium/locales`